hudi-client/hudi-flink-client/pom.xml

```diff
@@ -146,14 +146,6 @@
             <version>${hbase.version}</version>
             <scope>test</scope>
             <exclusions>
-                <exclusion>
-                    <groupId>org.codehaus.jackson</groupId>
-                    <artifactId>jackson-mapper-asl</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.codehaus.jackson</groupId>
-                    <artifactId>jackson-core-asl</artifactId>
-                </exclusion>
```

Comment:
this is exclusion, not adding the lib

Reply:
Thank you for your help to review the code, this should be caused by the conflict when merging master, I will modify the code.

Reply:
@slfan1989 what i meant was, why do you remove the exclusion here?